packages/babel-runtime/helpers/esm/package.json

```diff
@@ -1,3 +0,0 @@
-{
-  "type": "module"
-}
```

Comment:
Shouldn’t this file remain, since the files in this folder are **ECMAScript** modules?